tests/test_mcmc_helper.py

```diff
@@ -43,27 +46,109 @@ def test_mcmc_helper_parameters():
 
 
 def test_mcmc_helper_target_function():
+    """ Set up priors on the model parameters such that we can
+    readily compute their expected values. """
     data = build_data()
     model = build_model(data)
 
+    prior_width = 200
```

Comment:
```suggestion
    prior_width = 200.0
```
(It's used as a float)